ras_backstage/authentication/token_decoder.py

```diff
@@ -0,0 +1,19 @@
+import jwt
+from flask import current_app
+
+
+def decode_access_token(access_token):
+    decoded_jwt = jwt.decode(
```

Comment:
Decode can throw a DecodeError, shouldn't this be handled?
https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jwt.py#L83

Reply:
yes